web/beacon-app/.gitignore

```diff
@@ -11,6 +11,8 @@ node_modules
 dist
 dist-ssr
 *.local
+.yarn
```

Comment:
I had to run `yarn install` and these files were created, so I added them to the `.gitignore`